# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/TransportSecurityInfo.h
# Commit: fa57574775b2
# Full Hash: fa57574775b29f4a03667d9c350a1be269adfaf7
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-09-26 21:35:42
# Regressor Bug: 1577643
# File Overlap Count: 1
# Description:
#   Bug 1577643 - Implement a security info class for the quic transport. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44073
# ==============================================================================

diff -r 02f72f7040e4 -r fa57574775b2 security/manager/ssl/TransportSecurityInfo.h
--- a/security/manager/ssl/TransportSecurityInfo.h	Thu Sep 26 10:14:53 2019 +0000
+++ b/security/manager/ssl/TransportSecurityInfo.h	Wed Sep 25 20:23:56 2019 +0000
@@ -116,6 +116,7 @@
 
  protected:
   nsCOMPtr<nsIInterfaceRequestor> mCallbacks;
+  nsCOMPtr<nsIX509CertList> mSucceededCertChain;
 
  private:
   uint32_t mSecurityState;
@@ -127,7 +128,6 @@
   OriginAttributes mOriginAttributes;
 
   nsCOMPtr<nsIX509Cert> mServerCert;
-  nsCOMPtr<nsIX509CertList> mSucceededCertChain;
 
   /* Peer cert chain for failed connections (for error reporting) */
   nsCOMPtr<nsIX509CertList> mFailedCertChain;